jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -146,7 +149,11 @@ private static EventHandlers makeEventHandlers(
                             update.getProgress(), update.getUnfinishedAllocations()))));
   }
 
-  private static boolean isProgressFooterEnabled() {
+  private static boolean isProgressFooterEnabled(MavenSession session) {
```

Comment:
Now is a good time to add a test for this method. You'll need to make it package-private and add the `@VisibleForTesting`.

```java
@VisibleForTesting
tatic boolean isProgressFooterEnabled(...) {
```

Reply:
Taken care of in #1525.